README.md

```diff
@@ -0,0 +1,72 @@
+# StoreFront
+
+[![license](https://img.shields.io/github/license/mashape/apistatus.svg?style=flat-square)](https://choosealicense.com/licenses/mit/)
+[![API Reference](https://img.shields.io/badge/API_reference-latest-blue.svg?style=flat-square)](https://groupby.github.io/storefront)
+
+StoreFront (SF) is an e-commerce UI component library that is fully integrated with the GroupBy ecosystem/services.
+
+## Getting Started
+
+### Prerequisites
+
+ - yarn
+
+
+## Setup
+
+Use the `./scripts/setup.sh` script from the root of the project monorepo in order to build all of the StoreFront packages.
+
+```sh
+  ./scripts/setup.sh
+```
+
+## Commands
+
+Use the following commands from individual packages contained within the StoreFront monorepo. The individual packages can be within the [packages/@storefront](packages/@storefront) directory.
```

Comment:
Something to the effect of: "The following commands are run in the context of an individual package. These packages can be found within the ..."

Reply:
I see that this has been addressed, so I'll resolve. However, I left an additional comment below.